plugin-localization/src/main/java/com/mapbox/mapboxsdk/plugins/localization/LocalizationPlugin.java

```diff
@@ -277,11 +356,6 @@ private void convertExpressionV8(@NonNull MapLocale mapLocale, Layer layer, Prop
 
       String mapLanguage = mapLocale.getMapLanguage();
       if (!mapLanguage.equals(MapLocale.ENGLISH)) {
-        if (mapLanguage.equals(MapLocale.CHINESE)) {
-          // in streets v8 tiles chinese is declared as "name_zh-Hant" instead of "name_zh"
-          mapLanguage = MapLocale.CHINESE_V8;
```

Comment:
This is still needed, otherwise, we'll break the compatibility of `MapLocale.CHINESE` with `streets-v8` sources.

Reply:
This can be ignored if the proposed solution is implemented.

Reply:
With the current implementation, this is still needed.